tac/guidelines/project-consistency-guidelines.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: default
+title: Project Consistency
+parent: Guidelines
+grand_parent: LF Decentralized Trust TAC
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+This document outlines consistent approaches to certain aspects of running and maintaining an LFDT project.
+
+LFDT projects will always have their own approaches to how they manage and govern themselves, how they
+communicate and collaborate, and how they track and communicate progress.
+
+The LFDT already sets out `MUST` and `SHOULD` and `MAY` behaviours that projects should treat accordingly. This document
+is not intended to replace any of the existing governance rules set out by the LFDT. Instead it is designed improve
+consistency across projects such that inter-project collaboration is easier, and to make it easier for individuals to
+collaborate to multiple LFDT projects.
+
+**MRW** - There is potentially a lot of overlap with [project best practices](./project-best-practices.md)
+
+**MRW** - Perhaps this document should be wrapped into that document?
```

Comment:
@matthew1001 I don't feel strongly about this: My intuition for something called best practices is that it only contains things that are marked as `SHOULD` and `MAY` with all of those recommendations being at my discretion as a maintainer. If the project governance document will also contain `MUST` type items then to me it would make sense to instead wrap the best practices document into this one or just keep them separate with links to each other to avoid duplication. The reasoning behind this is that the name of the document (to me at least) implies whether it's recommendations only or recommendations plus rules that need to be followed.